src/server/svr_func.c

```diff
@@ -6435,12 +6440,18 @@ del_prov_vnode_entry(job *pjob)
 int
 action_backfill_depth(attribute *pattr, void *pobj, int actmode) {
 
-	if (pattr == NULL)
-		return PBSE_NONE;
+	if ( pobj == &server || pobj == dflt_scheduler) {
```

Comment:
This action should be same irrespective of which scheduler we are setting it on.